### PR TITLE
Fix dead links

### PR DESCRIPTION
**Pull request by Youssef1313**
The workflow fails due to dead links. Going to fix the links.

- The action seems to be running only in my fork. Probably GH Actions are disabled for the repo?
- It's still not passing due to few remaining issues.

### CODE REVIEW

File: release-notes/download-archive.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ This page provides an archive of previously released versions of the .NET Core r
 | Release Date | Description | Release Notes | |
 | :-- | :-- | :--: | :--: |
 | 2019/07/09 | 2.2.6 with SDK 2.2.108 | [release notes](2.2/2.2.6/2.2.6.md) | [download](2.2/2.2.6/2.2.6-download.md) |
-| 2019/07/09 | 2.2.6 with SDK 2.2.205 | - | [download](2.2/2.2.6/2.2.205-SDK-download.md) |
-| 2019/07/09 | 2.2.6 with SDK 2.2.301 | - | [download](2.2/2.2.6/2.2.301-SDK-download.md) |
-| 2019/05/21 | 2.2.5 with SDK 2.2.300 | - | [download](2.2/2.2.300/2.2.300-SDK-download.md) |
+| 2019/07/09 | 2.2.6 with SDK 2.2.205 | - | [download](2.2/2.2.6/2.2.205-sdk-download.md) |
+| 2019/07/09 | 2.2.6 with SDK 2.2.301 | - | [download](2.2/2.2.6/2.2.301-sdk-download.md) |
+| 2019/05/21 | 2.2.5 with SDK 2.2.300 | - | [download](2.2/2.2.300/2.2.300-sdk-download.md) |
```
Comment (Youssef1313) [inline release-notes/download-archive.md:12]:
```suggestion
| 2019/05/21 | 2.2.5 with SDK 2.2.300 | - | [download](2.2/2.2.300-SDK/2.2.300-SDK-download.md) |
```

File: release-notes/download-archive.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ This page provides an archive of previously released versions of the .NET Core r
 | Release Date | Description | Release Notes | |
 | :-- | :-- | :--: | :--: |
 | 2019/07/09 | 2.1.12 with SDK 2.1.508 (LTS) | [release notes](2.1/2.1.12/2.1.12.md) | [download](2.1/2.1.12/2.1.12-download.md) |
-| 2019/07/09 | 2.1.12 with SDK 2.1.605 (LTS) | - | [download](2.1/2.1.12/2.1.605-SDK-download.md) |
-| 2019/07/09 | 2.1.12 with SDK 2.1.701 (LTS) | - | [download](2.1/2.1.12/2.1.701-SDK-download.md) |
-| 2019/05/21 | 2.1.11 with SDK 2.1.700 (LTS) | - | [download](2.1/2.1.700/2.1.700-SDK-download.md) |
+| 2019/07/09 | 2.1.12 with SDK 2.1.605 (LTS) | - | [download](2.1/2.1.12/2.1.605-sdk-download.md) |
+| 2019/07/09 | 2.1.12 with SDK 2.1.701 (LTS) | - | [download](2.1/2.1.12/2.1.701-sdk-download.md) |
+| 2019/05/21 | 2.1.11 with SDK 2.1.700 (LTS) | - | [download](2.1/2.1.700/2.1.700-sdk-download.md) |
```
Comment (Youssef1313) [inline release-notes/download-archive.md:28]:
```suggestion
| 2019/05/21 | 2.1.11 with SDK 2.1.700 (LTS) | - | [download](2.1/2.1.700-SDK/2.1.700-SDK-download.md) |
```

**Comment by Youssef1313** [comment]
Need guidance for the remaining dead links:

```
Validating links in: /github/workspace/release-notes/download-archives/1.0-preview2-download.md.
##[error]Invalid link: 'dotnet-osx-x64.1.0.0.tar.gz' relative to '/github/workspace/release-notes/download-archives'.

Validating links in: /github/workspace/release-notes/1.1/README.md.
##[error]Invalid link: '1.1-supported-os.md' relative to '/github/workspace/release-notes/1.1'.

Validating links in: /github/workspace/release-notes/2.2/2.2.8/2.2.207-download.md.
##[error]Invalid link: '2.2.8-download.md' relative to '/github/workspace/release-notes/2.2/2.2.8'.

Validating links in: /github/workspace/release-notes/2.2/2.2.1/2.2.1.md.
##[error]Invalid link: '../2.1-known-issues.md' relative to '/github/workspace/release-notes/2.2/2.2.1'.

Validating links in: /github/workspace/release-notes/3.1/preview/3.1.0-preview3-install-instructions.md.
##[error]Invalid link: '3.1.0-preview3' relative to '/github/workspace/release-notes/3.1/preview'.

Validating links in: /github/workspace/release-notes/3.1/preview/3.1.0-preview2-install-instructions.md.
##[error]Invalid link: '3.1.0-preview2' relative to '/github/workspace/release-notes/3.1/preview'.

Validating links in: /github/workspace/release-notes/3.1/3.1.0/3.1.0-install-instructions.md.
##[error]Invalid link: '3.1.0' relative to '/github/workspace/release-notes/3.1/3.1.0'.

Validating links in: /github/workspace/release-notes/5.0/api-diff/netcoreapp3.1/.NET/standalone/5.0-standalone.md.
##[error]Invalid link: '5.0-standalone_System.CodeDom.md' relative to '/github/workspace/release-notes/5.0/api-diff/netcoreapp3.1/.NET/standalone'.
```

**Comment by mairaw** [comment]
@leecow @rbhanda is this something you can help @Youssef1313 with? He's trying to fix broken links in this repo.

**Comment by leecow** [comment]
@Youssef1313 - we're not using GitHub actions across the .NET repos yet. All of the link updates look good but we can't accept `workflows/markdown-links-verifier.yml` at this time.

**Comment by Youssef1313** [comment]
@leecow Sure, going to delete the yaml file. There are still some dead links that are good to fix if there are replacements.

**Comment by leecow** [comment]
LGTM

**Comment by Youssef1313** [comment]
@leecow Note: there are still a bunch of dead links: https://github.com/dotnet/core/pull/6065#issuecomment-798848080

**Pull request merged**